src/common/src/util/ordered/mod.rs

```diff
@@ -14,159 +14,4 @@
 
 mod serde;
 
-use std::cmp::Reverse;
-
-use OrderedDatum::{NormalOrder, ReversedOrder};
-
 pub use self::serde::*;
```

Comment:
maybe we should move it into `memcmp_encoding` and delete the `ordered` directory

Reply:
+1. They are a bit repetitive.